WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -163,6 +167,28 @@ extension EnvironmentValues {
     }
 }
 
+@available(iOS 17.0, *)
+private extension PointOfSaleDashboardView {
+    func enableCoupons() async {
```

Comment:
I didn't want to go too deep into where to add this function right now: Initially I wanted to add it to the coupons service but this means that if we want to trigger it from the dashboard it has to cross through aggregate model -> coupon controller -> coupon service. Since it's a store-wide setting, would it make sense to just add it to the aggregate model or the coupon controller?

Reply:
Understandable issue. I think one place it doesn't belong in is `PointOfSaleDashboardView`.

> Since it's a store-wide setting, would it make sense to just add it to the aggregate model or the coupon controller?

I agree that it's store-wide, so the aggregate model could work. However, I think we don't want to make it have too many responsibilities.

Since we have a coupon logic-related controller, moving it at least to a controller makes more sense.